openmc/universe.py

```diff
@@ -512,6 +512,19 @@ def get_all_cells(self, memo=None):
 
         return cells
 
+    def get_all_filled_cells(self):
+        """Return all cells that are contained within the universe that are
+        filled with a material
+
+        cells : collections.OrderedDict
+            Dictionary whose keys are cell IDs and values are :class:`Cell`
+            instances
+
+        """
+
+        return {id:cell for id, cell in self.get_all_cells().items() if cell.fill is not None}
```

Comment:
This doesn't return an OrderedDict, is that an intentional move away from OrderedDicts since we now support only later versions of python with ordered dictionaries?

Reply:
Ah sorry, I think as we are not using specific order dict methods like ```move_to_end()``` or ```.popitem()``` and python 3.7 is now the[ minimum version](https://github.com/openmc-dev/openmc/blob/d7fc8e59a27fd9581450b7ea25d89a7637c6cb03/setup.py#L64) we can use the normal dictionary? I based this on a [blog post on the topic](https://realpython.com/python-ordereddict/) but am happy with anything. For now I shall update the doc string to dict

Reply:
Yeah, no doubt we should migrate to dict as we don't use the additional OrderedDict methods anywhere (I think?), but we probably should just keep this return type to be consistent with the other similarly named methods and return an OrderedDict.